front/src/modules/activities/components/ActivityEditor.tsx

```diff
@@ -146,7 +146,7 @@ export function ActivityEditor({
   }
 
   return (
-    <StyledContainer>
+    <StyledContainer id="activity-editor-container">
```

Comment:
why are we adding an id here?

Reply:
This is the div that's actually being scrolled, this is how we're able to trigger the scroll. Please refer to the `handleFocus` function in https://github.com/twentyhq/twenty/pull/1102/files#diff-c44572409a28fab528d29546b936dea34183d1c5792efa54a838175d65197fb4